wdl/subworkflows/get_preseq_metrics.wdl

```diff
@@ -7,15 +7,17 @@ import "../tasks/preseq.wdl"
 
 workflow get_preseq_metrics {
     input {
+        Boolean? preseq_defects_mode
         File nuclear_bam
-        PreseqLcExtrapParams params = object {
-            defects: false,
-            extrap: "1.001e9",
-            step_size: "1e6"
-            }
         Resources resources
     }
 
+    PreseqLcExtrapParams params = object {
+        defects: select_first([preseq_defects_mode,false]),
```

Comment:
`preseq_defects_mode,false` space between these.

Reply:
Thanks. That linter.